.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy Storybook
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+jobs:
+  deploy:
+    name: Deploy Storybook
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Checkout GitHub Pages branch
+        run: |
+          git config user.name "GitHub Actions"
+          git config user.email "github-actions@github.com"
+          git checkout -B gh-pages
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
It looks like you have this install step twice. If the second copy isn't actually doing something useful, could you delete it?

Reply:
Ahaha woops, that was the result of sloppy copy/pasting on my part.